src/hw-monitor.h

```diff
@@ -331,6 +331,8 @@ namespace librealsense
                                       uint8_t * bufferToSend,
                                       int & length );
 
+         static command build_command_from_data(const std::vector<uint8_t> data);
```

Comment:
Indentation issue?

Reply:
Why do we need this static method? The only usage I see is in `hw_monitor_extended_buffers` which inherent from hw-monitor, no?

Reply:
Well, since this method does the inverse of the method fill_usb_buffer, and it is static, I thought we would want this to be too.
Initially, it was only a private method in the class. Let's speak about it

Reply:
> Indentation issue?

No - see above, same indentation

Reply:
> Indentation issue?

You are right, don't know why I saw that..